<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gconsignaciones</class>
 <widget class="QDialog" name="Gconsignaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Consignacion</string>
  </property>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Lista de consignaciones</string>
   </property>
   <widget class="QTableView" name="lista2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>114</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout2">
     <item>
      <widget class="QPushButton" name="addbutton2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Adicionar Nueva Consignacion</string>
       </property>
       <property name="statusTip">
        <string>Adicionar Nueva Consignacion</string>
       </property>
       <property name="text">
        <string>Adicionar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modificar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Modificar estado de una Consignacion</string>
       </property>
       <property name="statusTip">
        <string>Modificar estado de una Consignacion</string>
       </property>
       <property name="text">
        <string>Modificar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDockWidget" name="informaciondasignaradicionado">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>44</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="features">
        <set>QDockWidget::DockWidgetClosable</set>
       </property>
       <property name="windowTitle">
        <string>Informacion</string>
       </property>
       <widget class="QWidget" name="informacion2_2">
        <widget class="QLabel" name="informacionlabel2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Asignacion realizada 
 con exito</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="dasignarw">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Nueva Consignacion</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>211</width>
        <height>106</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Producto:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="productocomboBox2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label2_2">
         <property name="text">
          <string>Cantidad:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="cantidad2"/>
       </item>
      </layout>
     </widget>
     <widget class="QDockWidget" name="informacionproductoadicionado">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>524287</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetClosable</set>
      </property>
      <property name="windowTitle">
       <string>Producto Adicionado</string>
      </property>
      <widget class="QWidget" name="informacion2_4"/>
     </widget>
     <widget class="QTableView" name="listadasignar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>281</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addproductoDasignar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar Producto</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label2_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label2_3">
         <property name="text">
          <string>Consignatario:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="consignatariocombo2"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addasignar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
      </property>
     </widget>
     <widget class="QSpinBox" name="id2spinbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>47</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="estadolabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Estado:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="realizadoradio">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>92</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Realizado</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="pendienteradio">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>92</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pendiente</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="codven2spinbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>47</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="costo2spinbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="total2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelar">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/dialog-cancel.png</normaloff>:/images/dialog-cancel.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="consignatario">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="vendidobutton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Vendido</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/dialog-ok.png</normaloff>:/images/dialog-ok.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pendiente">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Pendiente</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/window-close.png</normaloff>:/images/window-close.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminar">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="ventaid_6">
        <property name="text">
         <string>Id. Consignacion:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dasignarid">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ventaid_3">
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="dasignarfecha">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ventaid_7">
        <property name="text">
         <string>Total en bs:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="dasignartotal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ventaid_5">
        <property name="text">
         <string>Consignatario:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="dasignarconsignatario">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ventaid_8">
        <property name="text">
         <string>Estado:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="estado">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="dasignardetalle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>301</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>331</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="volverboton">
      <property name="toolTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="statusTip">
       <string>Volver a la Pantalla Principal</string>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
      <property name="icon">
       <iconset resource="pandasoft.qrc">
        <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pandasoft.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>volverboton</sender>
   <signal>clicked()</signal>
   <receiver>Gconsignaciones</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
